internal/annotators/http/pki.go

```diff
@@ -25,13 +25,18 @@ import (
 	"strings"
 
 	"github.com/project-alvarium/alvarium-sdk-go/internal/annotators"
+	handler "github.com/project-alvarium/alvarium-sdk-go/internal/annotators/http/handler"
 	"github.com/project-alvarium/alvarium-sdk-go/internal/signprovider"
 	"github.com/project-alvarium/alvarium-sdk-go/internal/signprovider/ed25519"
 	"github.com/project-alvarium/alvarium-sdk-go/pkg/config"
 	"github.com/project-alvarium/alvarium-sdk-go/pkg/contracts"
 	"github.com/project-alvarium/alvarium-sdk-go/pkg/interfaces"
 )
 
+const (
+	testRequest string = "testRequest"
```

Comment:
If I understand the logic correctly, this key is used to reference the value within the incoming `Context` that corresponds to the request we need to validate. In that case, an external client application will need to set it so it probably needs to be in `/pkg/contracts/constants.go`. Right now, it seems very tightly coupled to the test `TestHttpPkiAnnotator_Do`. Rename to something like `HttpRequestKey`.